scss/_global.scss

```diff
@@ -43,6 +43,7 @@ $medium-gray: #cacaca !default;
 /// Color used for dark gray UI items.
 /// @type Color
 $dark-gray: #8a8a8a !default;
+$dark-gray: #8a8a8a !default;
```

Comment:
Duplicate

Reply:
Was not awara that I hit the shortcut to duplicate it.